deps/v8/src/api.cc

```diff
@@ -937,6 +937,8 @@ void Template::Set(v8::Local<Name> name, v8::Local<Data> value,
   i::Isolate* isolate = templ->GetIsolate();
   ENTER_V8(isolate);
   i::HandleScope scope(isolate);
+  auto value_obj = Utils::OpenHandle(*value);
```

Comment:
I landed this in https://codereview.chromium.org/1839983002/ but then reverted it because it broke our node CI